### PR TITLE
GH-21 Add support to reloadable holograms using command.

### PR DESCRIPTION
**Pull request by vLuckyyy**

### CODE REVIEW

File: src/main/java/com/eternalcode/lobbyheads/reload/ReloadService.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    public ReloadService(HologramService hologramService, ConfigurationService configurationService) {
+        this.hologramService = hologramService;
+        this.configurationService = configurationService;
+    }
+
+    public void reload() {
+        this.configurationService.reload();
+        this.hologramService.updateHolograms();
+    }
```
Comment (imDMK) [inline src/main/java/com/eternalcode/lobbyheads/reload/ReloadService.java]: This method should be moved to HologramService.
And if you want it to stay that way, I suggest changing the name to HologramReloadService.

**Review by imDMK — CHANGES_REQUESTED** [review]

**Review by rchomczyk — APPROVED** [review]

File: src/main/java/com/eternalcode/lobbyheads/reload/ReloadService.java (hunk 1 above)
Comment (vLuckyyy) [reply src/main/java/com/eternalcode/lobbyheads/reload/ReloadService.java]: This method does not only reload holograms, it also reloads the config and holograms.

Comment (imDMK) [reply src/main/java/com/eternalcode/lobbyheads/reload/ReloadService.java]: So I suggest changing the name to HologramReloadService.

Comment (vLuckyyy) [reply src/main/java/com/eternalcode/lobbyheads/reload/ReloadService.java]: > So I suggest changing the name to HologramReloadService.

Why? this reload config and holograms, not only holograms.

Comment (Rollczi) [reply src/main/java/com/eternalcode/lobbyheads/reload/ReloadService.java]: I suggest to remove hologram and config dependencies from this service and create interface Reloadable that hologram and config services will implements.

**Review by Rollczi — CHANGES_REQUESTED** [review]
Remove dependencies from reload service... See comment

**Review by imDMK — APPROVED** [review]

**Review by Rollczi — APPROVED** [review]

**Pull request merged**